android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -284,25 +360,62 @@ public void onError(String info) {
-        this.context = new WeakReference<>(context);
-        this.remoteDisplayClass = remoteDisplayClass;
-        int majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+    @Deprecated
+    public void startRemoteDisplayStream(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, VideoStreamingParameters parameters, final boolean encrypted){
+        configureGlobalParameters(context, remoteDisplayClass, isEncrypted, null);
         boolean isCapabilitySupported = internalInterface.getSystemCapabilityManager() != null && internalInterface.getSystemCapabilityManager().isCapabilitySupported(SystemCapabilityType.VIDEO_STREAMING);
         if (majorProtocolVersion >= 5 && !isCapabilitySupported) {
             DebugTool.logError(TAG, "Video streaming not supported on this module");
             stateMachine.transitionToState(StreamingStateMachine.ERROR);
             return;
         }
-        // regardless of VideoStreamingParameters are specified or not, we should refer to VideoStreamingCapability.
+        if (!HMILevel.HMI_NONE.equals(currentOnHMIStatus.getHmiLevel()) && VideoStreamManager.this.parameters == null) {
+            getVideoStreamingParams();
+        }
+        checkState();
+        processCapabilitiesWithPendingStart(encrypted, parameters);
+    }
+
+    private void configureGlobalParameters(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, boolean encrypted, List<VideoStreamingRange> listOfStreamingRange) {
+        this.context = new WeakReference<>(context);
+        this.remoteDisplayClass = remoteDisplayClass;
+        this.isEncrypted = encrypted;
+        this.majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+        if (listOfStreamingRange != null) {
+            this.listOfStreamingRanges = listOfStreamingRange;
+        }
+    }
+
+
+    private void processCapabilitiesWithPendingStart(boolean encrypted, VideoStreamingParameters parameters) {
+        final VideoStreamingParameters params = (parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
         if (majorProtocolVersion >= 5) {
             if (internalInterface.getSystemCapabilityManager() != null) {
-                final VideoStreamingParameters params = ( parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
                 internalInterface.getSystemCapabilityManager().getCapability(SystemCapabilityType.VIDEO_STREAMING, new OnSystemCapabilityListener() {
                     @Override
                     public void onCapabilityRetrieved(Object capability) {
-                        params.update((VideoStreamingCapability) capability, vehicleMake);    //Streaming parameters are ready time to stream
-                        startStreaming(params, encrypted);
+                        VideoStreamingParameters params = new VideoStreamingParameters();
```

Comment:
```suggestion

```
By removing this line the params created at the begining of processCapabilitiesWithPendingStart will be used and then will be updated based on the capability retrieved